modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-report.adoc

```diff
@@ -26,6 +26,11 @@ The command collects information about the system and packages everything in an
 If you specify `all`, everything is included.
 You can also finetune the selection by passing classifiers to the tool, e.g `logs tx threads`.
 
+Starting from Neo4j version 5.15, the Neo4j Admin report tool allows you to choose the databases for which you want to generate a report.
```

Comment:
All this looks good, but I'm wondering if we should reword it slightly. Most of the report consist of dbms level things, that will be collected no matter what (depending on sent in specifiers). Maybe something like
`...allows you to choose the databases for which you want to included database specific information.
You can generate a report covering all databases...`

I'm not sure, what do you think?, my point is just that many of the collected things are not for a single db and will be collected independent of the filter

Reply:
Sounds good to me. I'll reword it.